### PR TITLE
62 basic layout of comparision widget

### PR DESCRIPTION
**Pull request by qzyan**
- update the Comparison component 
- update the RelatedProducts component
- update the YourProduct component 
- update the Card component

### CODE REVIEW

File: client/src/components/App/App.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
+.product-overview {
```
Comment (hepnerthomas) [inline client/src/components/App/App.css:1]: @qzyan Are we not using the normalize.css anymore for the App?

File: client/src/components/Comparison/Card/Card.jsx [hunk 2]
```diff
@@ -1,12 +1,20 @@
 import React from 'react';
 import card from './Card.module.css';
 
+// eslint-disable-next-line react/prefer-stateless-function
```
Comment (hepnerthomas) [inline client/src/components/Comparison/Card/Card.jsx:4]: @qzyan What's this comment for?

File: client/src/components/Comparison/Comparison.jsx [hunk 3]
```diff
@@ -1,13 +1,16 @@
 import React from 'react';
-import comparison from './Comparison.module.css';
-import Card from './Card/Card.jsx';
+// eslint-disable-next-line import/extensions
```
Comment (hepnerthomas) [inline client/src/components/Comparison/Comparison.jsx:2]: what are these comments for?

File: client/src/components/Comparison/RelatedProducts/RelatedProducts.jsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+class RelatedProducts extends React.Component {
+  render() {
+    return (
+      <div className="related-products">
+        <div>
+          RELATED PRODUCTS
+        </div>
+        <div className={relatedProducts.main}>
+          <div className={relatedProducts.container}>
+          <Card />
```
Comment (hepnerthomas) [inline client/src/components/Comparison/RelatedProducts/RelatedProducts.jsx:14]: Why are you rendering `<Card />` multiple times? Is this just a placeholder?

File: client/src/components/Comparison/YourProducts/YourProducts.jsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+class YourProducts extends React.Component {
+  render() {
+    return (
+      <div className="your-products">
+        <div>
+          RELATED PRODUCTS
+        </div>
+        <div className={yourProducts.main}>
+          <div className={yourProducts['card-container']}>
+            <Card />
```
Comment (hepnerthomas) [inline client/src/components/Comparison/YourProducts/YourProducts.jsx:14]: same - lots of different `<Card />` placeholders here

**Review by hepnerthomas — CHANGES_REQUESTED** [review]
@qzyan Please reply to comments just so I understand your thinking. Happy to approve after that!

File: client/src/components/App/App.css (hunk 1 above)
Comment (qzyan) [reply client/src/components/App/App.css:1]: This is the CSS code Allen shared in the example CSS file. Use it as a skeleton of the whole app.

File: client/src/components/Comparison/Card/Card.jsx (hunk 2 above)
Comment (qzyan) [reply client/src/components/Comparison/Card/Card.jsx:4]: I will add a state to this component later. Now just disable the lint warning for this  stateless-function check

File: client/src/components/Comparison/Comparison.jsx (hunk 3 above)
Comment (qzyan) [reply client/src/components/Comparison/Comparison.jsx:2]: I will add a state to this component later. Now just disable the lint warning for this  stateless-function check

File: client/src/components/Comparison/RelatedProducts/RelatedProducts.jsx (hunk 4 above)
Comment (qzyan) [reply client/src/components/Comparison/RelatedProducts/RelatedProducts.jsx:14]: Put them as placeholders to hold the actual products later

File: client/src/components/Comparison/YourProducts/YourProducts.jsx (hunk 5 above)
Comment (qzyan) [reply client/src/components/Comparison/YourProducts/YourProducts.jsx:14]: Put them as placeholders to hold the actual products later

**Comment by hepnerthomas** [comment]
looks good - merging now! @qzyan

**Pull request merged**